java/server/src/org/openqa/selenium/grid/data/NodeStatus.java

```diff
@@ -182,13 +194,15 @@ public static NodeStatus fromJson(JsonInput input) {
       maxSessions,
       stereotypes,
       sessions,
+      draining,
       registrationSecret);
   }
 
   public static class Active {
 
     private final Capabilities stereotype;
     private final SessionId id;
+    private final boolean isDraining = true;
```

Comment:
Nodes are draining by default. Better to leave this as the default `false` value, no?